src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./stars.scss";
+@import "./card.scss";
+@import "./_variables.scss";
```

Comment:
The _variables.scss file should be imported first as it may contain variables that are used in other SCSS files.